src/xb-builder-node.c

```diff
@@ -297,6 +301,41 @@ xb_builder_node_parse_literal_text (XbBuilderNode *self, const gchar *text, gssi
 	return g_string_free (tmp, FALSE);
 }
 
+/**
+ * xb_builder_node_tokenize_text:
+ * @self: a #XbBuilderNode
+ *
+ * Tokenize text added with xb_builder_node_set_text().
+ *
+ * Since: 0.3.1
+ **/
+void
+xb_builder_node_tokenize_text (XbBuilderNode *self)
+{
+	XbBuilderNodePrivate *priv = GET_PRIVATE (self);
+	g_auto(GStrv) tokens = NULL;
+	g_auto(GStrv) ascii_tokens = NULL;
+
+	g_return_if_fail (XB_IS_BUILDER_NODE (self));
+
+	if (priv->text == NULL)
+		return;
+	tokens = g_str_tokenize_and_fold (priv->text, NULL, &ascii_tokens);
```

Comment:
Might want to hook `translit_locale` up to an argument in `xb_builder_node_tokenize_text()`, as I imagine the caller might have information about the locale of the source string in some cases.

Reply:
hmm, maybe the galaxy brain thing to do would be to use the `xml:lang` attribute -- this makes it magically just work with AppStream...

Reply:
Yes, that works.